student.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
   attr_accessor :classroom
```

Comment:
- [ ] Kindly add a method that will link a student to a classroom. A student belongs to a classroom.